pkg/runtime/OWNERS

```diff
@@ -1,5 +1,41 @@
-assignees:
-  - caesarxuchao
-  - deads2k
-  - lavalamp
-  - smarterclayton
+approvers:
+- caesarxuchao
+- deads2k
+- lavalamp
+- smarterclayton
+reviewers:
```

Comment:
Instead of telling you who to remove, please delete this list and include:

- lavalamp
- smarterclayton
- wojtek-t
- deads2k
- derekwaynecarr
- caesarxuchao
- mikedanese
- nikhiljindal
- gmarek
- krousey
- timothysc
- piosz
- mbohlool